gradle.properties

```diff
@@ -1,5 +1,5 @@
 group = org.threadly
 version = 0.1.0-SNAPSHOT
-threadlyVersion = 3.8.0
-litesocketsVersion = 2.0.0
+threadlyVersion = 4.1.0
```

Comment:
In theory threadly should not need to be specified, it should be picked up from litesockets.